src/system/logger.hh

```diff
@@ -0,0 +1,317 @@
+/*!
+ *
+﻿ * Copyright (C) 2015 Technical University of Liberec.  All rights reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License version 3 as published by the
+ * Free Software Foundation. (http://www.gnu.org/licenses/gpl-3.0.en.html)
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ *
+ * @file    logger.hh
+ * @brief
+ */
+
+#ifndef LOGGER_HH_
+#define LOGGER_HH_
+
+
+#include <iostream>
+#include <sstream>
+#include <fstream>
+#include <vector>
+#include <mpi.h>
```

Comment:
Remove mpi and time_point includes:
- move TimePoint member form Logger into LoggerOptions
- move LoggerOptions into separate file.